ui/src/app/pools/views/PoolAdd/PoolAdd.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import PoolForm from "../PoolForm";
+
+export const PoolAdd = () => {
+  return <PoolForm />;
+};
+
```

Comment:
This component is a bit anaemic, but will make more sense once PoolEdit is added in a followup.